staging/cse/windows/provisioningscripts/cleanupnetwork.ps1

```diff
@@ -1,13 +1,8 @@
 $Global:ClusterConfiguration = ConvertFrom-Json ((Get-Content "c:\k\kubeclusterconfig.json" -ErrorAction Stop) | out-string)
 
 $global:NetworkMode = "L2Bridge"
-$global:ContainerRuntime = $Global:ClusterConfiguration.Cri.Name
 $global:NetworkPlugin = $Global:ClusterConfiguration.Cni.Name
-$global:HNSModule = "c:\k\hns.psm1"
```

Comment:
Remove the outdated comments at the same time: https://github.com/Azure/AgentBaker/blob/ae6f7253fd6b2d813106e67ac3c0ad9b300a4283/staging/cse/windows/azurecnifunc.ps1#L458

Reply:
I still can search `docker`, `useContainerD` under `staging\cse\windows`
![image](https://github.com/Azure/AgentBaker/assets/62196586/1239d8ad-e4fb-42df-af44-ebbceb93d7b0)